test/FunctionalTests/Microsoft.OData.Core.Tests/NavigationPropertyOnComplexTests.cs

```diff
@@ -1102,7 +1199,7 @@ public void AutoComputeIdForNavUnderComplex()
             // itemsList[3] is the nested info for complex property WorkAddress, skip the validation.
 
             // City under Address
-            nestInfo = itemsList[4];
+            nestInfo = itemsList[6];

```

Comment:
>[6]; [](start = 32, length = 4)

add comments for item 4 and 5 if you want to skip the validation for them?

Reply:
#Resolved